### PR TITLE
update service state update tasks to async APIs

### PR DESCRIPTION
**Pull request by WangLiNaruto**
Fixed eclipse-xpanse/xpanse#1613
image
image
image
image
image
image
image
image
image

### CODE REVIEW

**Comment by Alice1319** [comment]
LGTM

File: src/components/content/order/serviceState/stop/StopServiceStatusAlert.tsx [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+        } else {
+            errorMessage = serviceStateStopQuery.error.message;
+        }
+        deployedService.serviceState = DeployedService.serviceState.RUNNING;
+        return (
+            <div className={'submit-alert-tip'}>
+                {' '}
+                <Alert
+                    message={errorMessage}
+                    description={<OrderSubmitResultDetails msg={'Stop request failed'} uuid={deployedService.id} />}
```
Comment (swaroopar) [inline src/components/content/order/serviceState/stop/StopServiceStatusAlert.tsx:74]: msg - Service stop request failed

File: src/components/content/order/serviceState/stop/StopServiceStatusAlert.tsx [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+                <div className={'submit-alert-tip'}>
+                    {' '}
+                    <Alert
+                        message={
+                            serviceStopQuery.data.latestRunningManagementTask.taskType +
+                            ' ' +
+                            serviceStopQuery.data.latestRunningManagementTask.taskStatus
+                        }
+                        description={
+                            <OrderSubmitResultDetails msg={'Service stop successfully'} uuid={deployedService.id} />
```
Comment (swaroopar) [inline src/components/content/order/serviceState/stop/StopServiceStatusAlert.tsx:155]: msg - Service stopped successfully

File: src/components/content/order/serviceState/restart/RestartServiceStatusAlert.tsx [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+                <div className={'submit-alert-tip'}>
+                    {' '}
+                    <Alert
+                        message={
+                            serviceRestartQuery.data.latestRunningManagementTask.taskType +
+                            ' ' +
+                            serviceRestartQuery.data.latestRunningManagementTask.taskStatus
+                        }
+                        description={
+                            <OrderSubmitResultDetails msg={'Service restart successfully'} uuid={deployedService.id} />
```
Comment (swaroopar) [inline src/components/content/order/serviceState/restart/RestartServiceStatusAlert.tsx:154]: msg - Service restarted successfully

File: src/components/content/order/serviceState/start/StartServiceStatusAlert.tsx [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+                <div className={'submit-alert-tip'}>
+                    {' '}
+                    <Alert
+                        message={
+                            serviceStartQuery.data.latestRunningManagementTask.taskType +
+                            ' ' +
+                            serviceStartQuery.data.latestRunningManagementTask.taskStatus
+                        }
+                        description={
+                            <OrderSubmitResultDetails msg={'Service start successfully'} uuid={deployedService.id} />
```
Comment (swaroopar) [inline src/components/content/order/serviceState/start/StartServiceStatusAlert.tsx:154]: msg - Service started successfully

**Review by swaroopar — CHANGES_REQUESTED** [review]

File: src/components/content/order/serviceState/restart/RestartServiceStatusAlert.tsx [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+            'details' in serviceStateRestartQuery.error.body
+        ) {
+            const response: Response = serviceStateRestartQuery.error.body as Response;
+            errorMessage = response.details;
+        } else {
+            errorMessage = serviceStateRestartQuery.error.message;
+        }
+        deployedService.serviceState = DeployedService.serviceState.RUNNING;
+        return (
+            <div className={'submit-alert-tip'}>
```
Comment (swaroopar) [inline src/components/content/order/serviceState/restart/RestartServiceStatusAlert.tsx:60]: we must migrate all CSS classes to use CSS Modules. Please refer to other classes which are already using it. or JinPing can also help you with it.

Comment (swaroopar) [inline src/components/content/order/serviceState/restart/RestartServiceStatusAlert.tsx:60]: no space between alert and the button below. Please make it consistent with all other alerts in the myservices UI.

image

**Review by swaroopar — CHANGES_REQUESTED** [review]

**Review by swaroopar — APPROVED** [review]
LGTM

**Pull request merged**